### DISCUSSION TITLE
Does renovate collect any diagnostic information?

### DISCUSSION THREAD

**Discussion by justinas-b** [renovatebot/renovate#20848] [Request Help] [answered]
### Type of discussion.

I just want to chat

### Tell us more.

Hey there, we are running a self-hosted version of renovate and would like to know / get confirmation if renovate collects any kind of information, like diagnostic one? I mean, when we run renovate, is any kind of information being transmitted to remote servers?

**Comment by HonkingGoose** [comment, marked as answer]
We have a page in the docs about security and permissions see: https://docs.renovatebot.com/security-and-permissions/

Related discussion:

- https://github.com/renovatebot/renovate/discussions/19553

**Comment by rarkins** [comment]
Renovate does not "phone home" with any diagnostics or telemetry. See #19553 for a deeper discussion

**Comment by justinas-b** [comment]
Thanks both for really quick answer 🙏
